awsbestpractices/bad-params/AwsHttpClient.java

```diff
@@ -0,0 +1,33 @@
+package example;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBScanExpression;
+
+public class AwsHttpClientTestCases {
+
+    public void awsHttpClient() {
+        AmazonDynamoDBClient dynamoDBClient = new AmazonDynamoDBClient(credentialsProvider, clientConfiguration);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Client constructors are now deprecated in favor of using builders to create the client. Consider using the builder implementation instead.

[Learn more](https://aws.amazon.com/blogs/developer/client-constructors-now-deprecated/)